script/test/all

```diff
@@ -38,14 +38,16 @@ for version in $DOCKER_VERSIONS; do
 
   trap "on_exit" EXIT
 
-  repo="dockerswarm/dind"
+  repo="docker"
 
   docker run \
     -d \
     --name "$daemon_container" \
     --privileged \
     --volume="/var/lib/docker" \
-    "$repo:$version" \
+    -v ~/.docker/config.json:/code/.docker/config.json \
+    -e "DOCKER_TLS_CERTDIR=" \
+    "$repo:$version-dind" \
```

Comment:
The `$repo` variable is only used here; perhaps remove the variable above, and just inline it here;

```suggestion
    "docker:$version-dind" \
```